deploy/crds/openebs.io_blockdevices_crd.yaml

```diff
@@ -0,0 +1,247 @@
+                    kind:
+                      description: Kind is the type of link like by-id or by-path.
+                      enum:
+                      - by-id
+                      - by-path
+                      type: string
+                    links:
+                      description: Links are the soft links
+                      items:
+                        type: string
+                      type: array
+                  type: object
+                type: array
+              filesystem:
+                description: FileSystem contains mountpoint and filesystem type
+                properties:
+                  fsType:
+                    description: Type represents the FileSystem type of the block
+                      device
+                    type: string
+                  mountPoint:
+                    description: MountPoint represents the mountpoint of the block
+                      device.
+                    type: string
+                type: object
+              nodeAttributes:
+                description: NodeAttributes has the details of the node on which BD
+                  is attached
+                properties:
+                  nodeName:
+                    description: NodeName is the name of the Kubernetes node resource
+                      on which the device is attached
+                    type: string
+                required:
+                - nodeName
+                type: object
+              parentDevice:
+                description: "ParentDevice was intended to store the UUID of the parent
+                  Block Device as is the case for partitioned block devices. \n For
+                  example: /dev/sda is the parent for /dev/sda1 To be deprecated"
+                type: string
+              partitioned:
+                description: Partitioned represents if BlockDevice has partitions
+                  or not (Yes/No) Currently always default to No. To be deprecated
+                enum:
+                - "Yes"
+                - "No"
+                type: string
+              path:
+                description: Path contain devpath (e.g. /dev/sdb)
+                type: string
+            required:
+            - capacity
+            - details
+            - devlinks
+            - nodeAttributes
+            - partitioned
+            - path
+            type: object
+          status:
+            description: DeviceStatus defines the observed state of BlockDevice
+            properties:
+              claimState:
+                description: ClaimState represents the claim state of the block device
+                enum:
+                - Claimed
+                - Unclaimed
+                - Released
+                type: string
+              state:
+                description: State is the current state of the blockdevice (Active/Inactive)
+                enum:
+                - Active
+                - Inactive
```

Comment:
`Unknown` is also a valid state for a BlockDevice. When the NDM daemon on the node goes down, the blockdevices will be in an unknown state.